components/collector/src/source_collectors/bandit/security_warnings.py

```diff
@@ -28,6 +31,6 @@ def _parse_json(self, json: JSON, filename: str) -> Entities:
 
     def _include_entity(self, entity: Entity) -> bool:
         """Return whether to include the warning in the measurement."""
-        return entity["issue_severity"].lower() in self._parameter("severities") and entity[
-            "issue_confidence"
-        ].lower() in self._parameter("confidence_levels")
+        return super()._include_entity(entity) and entity["issue_confidence"].lower() in self._parameter(
```

Comment:
Is it more readable when moving `super()._include_entity(entity)` to the back of the line, or forcing a linebreak after `and` ?

Reply:
I'll add a helper method for the issue confidence. That way we get something that fits on one line.